drivers/sdhc/Kconfig.sdhc

```diff
@@ -7,3 +7,11 @@ config SDHCI_GENERIC
 	help
 	  SD Host Controller (SDHC) generic implementation module according to
 	  SD Host Controller Standard Specification (Version 3.0).
+
+config SDHCI_GENERIC_ADMA2_DESC_MAX_LEN
+	int "ADMA2 max xfer size per descriptor"
+	default 65532
```

Comment:
You can limit the minimum and maximum number for an integer option by adding the `range` property:
`range 1 65536`

Reply:
fixed